at-quannguyen/W4/controllers/session_controller.rb

```diff
@@ -0,0 +1,71 @@
+require_relative '../config/db'
+require 'bcrypt'
+$session = {}
+
+class SessionController
+  def initialize
+    @db = Db.new
+    @connect = @db.connect
+    @db.select_db
+  end
+
+  def login(obj)
+    if user_password_by_employee(obj.username, obj.password)
+      $session[:username] = obj.username
+    else
+      return false
+    end
+    true
+  end
+
+  def logout
+    if $session.key?(:username)
+      $session.delete(:username)
+      return true
+    end
+    false
+  end
+
+  def register(obj)
+    if username_by_employee(obj.username).count.zero?
+      obj.password = SessionController.digest(obj.password)
+      statement = @connect.prepare('INSERT INTO employee (name, role, username, password, leave_balance) VALUES (?, ?, ?, ?, ?)')
+      statement.execute(obj.name, obj.role, obj.username,
+                        obj.password, obj.leave_balance)
+      return true
+    end
+    false
+  end
+
+  def current_user
+    @current_user ||= username_by_employee($session[:username])
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7deb92b5e1c7275222ca44a289eebcc27cbc38fe/at-quannguyen/W4/controllers/session_controller.rb#L41"></span>Style/GlobalVars: Do not introduce global variables. (https://github.com/rubocop-hq/ruby-style-guide#instance-vars, http://www.zenspider.com/Languages/Ruby/QuickRef.html)</td>
    </tr>
  </tbody>
</table>